onchain/src/nip01.cairo

```diff
@@ -0,0 +1,12 @@
+#[derive(Copy, Drop)]
+struct NIP01Event {
+    id: u256,
+    pubkey: u256,
+    created_at: u64,
+    kind: u16,
+    tags: Array<ByteArray>,
```

Comment:
`tags` should be an array of arrays of `ByteArray`s.

Reply:
Alright